### PR TITLE
feat: add new mainnet and tnet bootstrap peers

### PR DESCRIPTION
**Pull request by nathanielc**

### CODE REVIEW

File: one/src/lib.rs [hunk 1]
```diff
@@ -214,20 +214,17 @@ impl Network {
...
-                "/dns/rust-ceramic-v4-tnet-1.3box.io/tcp/4101/p2p/12D3KooWPYNbc6VBfPuJQ84sNb4xNXysD383ZXZuLYbG8xTmHmfj/p2p/12D3KooWPYNbc6VBfPuJQ84sNb4xNXysD383ZXZuLYbG8xTmHmfj".to_string(), // cspell:disable-line
+                "/dns4/bootstrap-tnet-rust-ceramic-1.3box.io/tcp/4101/p2p/12D3KooWMqCFj5bnwuNi6D6KLhYiK4C8Eh9xSUKv2E6Jozs4nWEE",
+                "/dns4/bootstrap-tnet-rust-ceramic-2.3box.io/tcp/4101/p2p/12D3KooWPFGbRHWfDaWt5MFFeqAHBBq3v5BqeJ4X7pmn2V1t6uNs",
             ],
             Network::DevUnstable => vec![
-                "/dns4/go-ipfs-ceramic-public-qa-external.3boxlabs.com/tcp/4011/ws/p2p/QmPP3RdaSWDkhcxZReGo591FWanLw9ucvgmUZhtSLt9t6D".to_string(),  // cspell:disable-line
-                "/dns4/go-ipfs-ceramic-private-qa-external.3boxlabs.com/tcp/4011/ws/p2p/QmXcmXfLkkaGbQdj98cgGvHr5gkwJp4r79j9xbJajsoYHr".to_string(),  // cspell:disable-line
-                "/dns4/go-ipfs-ceramic-private-cas-qa-external.3boxlabs.com/tcp/4011/ws/p2p/QmRvJ4HX4N6H26NgtqjoJEUyaDyDRUhGESP1aoyCJE1X1b".to_string(),  // cspell:disable-line
+                "/dns4/go-ipfs-ceramic-public-qa-external.3boxlabs.com/tcp/4011/ws/p2p/QmPP3RdaSWDkhcxZReGo591FWanLw9ucvgmUZhtSLt9t6D",
```
Comment (stbrody) [inline one/src/lib.rs:225]: do we not have c1 bootstrap nodes for dev-unstable?

Comment (stbrody) [reply one/src/lib.rs:225]: if not, should we just have *no* bootstrap nodes for dev-unstable?  Keeping the kubo ones around doesn't seem like it makes sense, once we decommission them we'll just wind up with a lot of connection errors filling the logs

**Review by stbrody — APPROVED** [review]

File: one/src/lib.rs [hunk 2]
```diff
@@ -214,20 +214,17 @@ impl Network {
...
+                "/dns4/go-ipfs-ceramic-private-qa-external.3boxlabs.com/tcp/4011/ws/p2p/QmXcmXfLkkaGbQdj98cgGvHr5gkwJp4r79j9xbJajsoYHr",
```
Comment (AaronGoldman) [inline one/src/lib.rs:226]: `4011`
did you mean 
`4101`

Comment (AaronGoldman) [reply one/src/lib.rs:226]: never mind spencer already pointed out it is still GO

**Pull request merged**